File: Problems/No.2427_Number_of_Common_Factors/number_of_common_factors.cpp
class Solution {
public:
    int commonFactors(int a, int b) {
        
        // parameter
        int CF_cnt = 1;

        // start computing
        for (int i=2; i<=std::min(a, b); i++) 
            if ((a%i == 0) && (b%i == 0)) CF_cnt++;

        return CF_cnt;

    }
};

File: Problems/No.2427_Number_of_Common_Factors/number_of_common_factors.py
class Solution(object):
    def commonFactors(self, a, b):
        """
        :type a: int
        :type b: int
        :rtype: int
        """

        # parameter
        CF_cnt = 1
    
        # start computing
        for i in range(2, min(a, b)+1):
            if ((a%i == 0) and (b%i == 0)):
                CF_cnt += 1
        
        return CF_cnt
